File: web3rpc/rpc-specs/code-samples/python/debug/blockchainInspection/startContractWarmUp.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

address = "0x4Cd39B49064a31E966cA0ddF4111aCe2eD7E9502"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.start_contract_warm_up(address)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/blockchainInspection/startContractWarmUp.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const address = "0x4Cd39B49064a31E966cA0ddF4111aCe2eD7E9502"

    provider.debug.startContractWarmUp(address, {}, (err, data, response) => {
        console.log(data);
    });
}
)()